src/kartikey/WebTableEmployeeManager.java

```diff
@@ -0,0 +1,71 @@
+package kartikey;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import kartikey.base.PredDefindActions;
+
+public class WebTableEmployeeManager extends PredDefindActions {
+	private WebDriver driver;
+	
+	WebDriver setup(String url) {
+	return	driver= start(url);
+	}
+	
+	String getUrl() {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Please provide the url");
+		String url = scanner.next();
+		return url;
+	}
+	void employeeAndManager(WebDriver driver) throws InterruptedException {
+		driver.findElement(By.xpath("//a[text()= 'Demo Tables']")).click();
+		Thread.sleep(3000);
+		List<WebElement> listOfManagerID= driver.findElements(By.xpath("//table[@class='table table-striped']/tbody/tr/td[4]"));
+		
+		/*LinkedHashMap<String, Integer>lhm=new LinkedHashMap<String, Integer>();
+		for(WebElement element:listOfManagerID) {
+			if(lhm.containsKey(element.getText())) 
+				lhm.put(element.getText(), lhm.get(element.getText())+1);
+			else
+				lhm.put(element.getText(), 1);		
+		}
+		System.out.println(lhm);*/
+//************************************<By Set>***********************************		
+		HashSet<String> hs= new HashSet<String>();
+		for(WebElement element:listOfManagerID) {
+			hs.add(element.getText());
+		}
+		//System.out.println(hs);		
+		int maxEmpManager=0;
+		String maxEmpManagerID="";
+		for(String str:hs) {			
```

Comment:
what is purpose of writing two loops

Reply:
hs has unique emplist & listOfManagerID have all details
then why we need to check two times
please update code

Reply:
Hello Harsh, Second loop is to get the maximum no. of employees that reports to manager, From first i get the Unique Managers and from the second i get how many employees are reporting to that particular manager.
I tried to fit in one loop but i m stuck as how to add+iterate+than re check to give count.. i need help will contact you tomorrow